migrations/2025-07-15-081447_group-follow/up.sql

```diff
@@ -1,6 +1,8 @@
 CREATE TABLE community_community_follow (
+    target_id int REFERENCES community (id) ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
     community_id int REFERENCES community (id) ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
-    follower_id int REFERENCES community (id) ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
-    PRIMARY KEY (community_id, follower_id)
+    PRIMARY KEY (target_id, community_id)
```

Comment:
Change the primary key to `(community_id, target_id)` so it's consistent with other action tables (e.g. person actions uses `(person_id, target_id)`). This will make confusion less likely when someone uses the `community_community_follow::table.find` method.

This will change which column needs its own index, so also change the index below to `(target_id)`.